src/app/batch_generate_keys/batch_generate_keys.ml

```diff
@@ -229,8 +227,16 @@ let output_there_and_back_cmds =
          else None
        in
        let batch_count = ref 0 in
+       let base_send_amount = Currency.Amount.of_formatted_string "2" in
+       let initial_send_amount =
+         Option.value_exn
+           (Currency.Amount.add base_send_amount
+              (Currency.Amount.of_fee Mina_base.Signed_command.minimum_fee))
```

Comment:
Nit: you probably want the account creation fee here.

Reply:
how much is that?